raiden/constants.py

```diff
@@ -33,6 +41,8 @@
 NULL_ADDRESS = to_checksum_address(NULL_ADDRESS_BYTES)
 
 EMPTY_HASH = BlockHash(bytes(32))
+EMPTY_BALANCE_HASH = BalanceHash(bytes(32))
+EMPTY_MESSAGE_HASH = AdditionalHash(bytes(32))
```

Comment:
why call this `EMPTY_MESSAGE_HASH`, but the type `AdditionalHash`

Reply:
The reason he did this is because message hash is actually types as `AdditionalHash` in the codebase. It's one of our nice naming confusions. There is no `MessageHash` type. I guess that it should not be the work of this PR to change it but it should be done separately if at all.